app/renderer/materialize.js

```diff
@@ -35,6 +35,9 @@ module.exports = {
             alignment: 'left'
         });
     },
+    toggleTap(selector, status) {
```

Comment:
Hi, I understand this method is to replace the $ in the other file, but I don't see this being used anywhere in the code

Reply:
you are right i introduced it as a fix to the file but after i reviewed the code from materializecss i've seen that close is not mandatory because when the tap is open the default click behaviour will close it

Reply:
Oh, nice, then it would be better to remove it